lib/checkPullRequestReview.js

```diff
@@ -287,6 +288,102 @@ const handlePullRequestReview = async (context) => {
   }
 };
 
+/**
+ * This function handles the case when a PR author comments on the pull
+ * request. If pr author comments on the PR asking reviewers to please
+ * take a look (PTAL).
+ * The function checks if the reviewers have been assigned and if not,
+ * assigns them.
+ *
+ * @param {import('probot').Context} context
+ */
+const handleResponseToReview = async (context) => {
+  /**
+   * @type {import('probot').Octokit.IssuesGetResponse}
+   */
+  const pullRequest = context.payload.issue;
+  const pullRequestProp = 'pull_request';
+  const isPullRequest = pullRequestProp in pullRequest;
+  if (!isPullRequest) {
+    return;
+  }
+
+  /**
+   * @type {import('probot').Octokit.IssuesGetCommentResponse}
+   */
+  const comment = context.payload.comment;
+  const commenter = comment.user.login;
+  const commenterIsAuthor = commenter === pullRequest.user.login;
+  if (!commenterIsAuthor) {
+    return;
+  }
+
+  const containsPtalComment = PTAL_COMMENTS.some((item) =>
+    comment.body.toLowerCase().includes(item.toLowerCase())
+  );
+  if (!containsPtalComment) {
+    return;
+  }
+
+  // Pause for 3 minutes in case author wants to perform
+  // the required actions.
+  await utilityModule.sleep(THREE_MINUTES);
+
+  // Fetch the pull request in case there has been any changes since
+  // the comment was made.
+  const pullRequestResponse = await context.github.pulls.get(
+    context.repo({
+      pull_number: pullRequest.number,
+    })
+  );
+  const updatedPullRequest = pullRequestResponse.data;
+
+  const usersInComment = utilityModule.getUsernamesFromText(comment.body);
+  // Check if assignment was carried out by author.
+  const assignees = updatedPullRequest.assignees.map(
+    (assignee) => assignee.login
+  );
+  const unassignedUsers = usersInComment.filter(
+    (user) => !assignees.includes(user)
+  );
+
+  if (unassignedUsers.length === 0) {
+    // All users in comment have been assigned.
+    return;
+  }
+
+  // Assign users in comment
+  context.github.issues.addAssignees(
+    context.repo({
+      issue_number: updatedPullRequest.number,
+      assignees: unassignedUsers,
+    })
+  );
+
+  // Unassign author if assigned.
+  const authorIsAssigned = updatedPullRequest.assignees.some(
+    (assignee) => assignee.login === updatedPullRequest.user.login
+  );
+  if (authorIsAssigned) {
+    context.github.issues.removeAssignees(
+      context.repo({
+        issue_number: updatedPullRequest.number,
+        assignees: [updatedPullRequest.user.login],
+      })
+    );
+
+    context.github.issues.createComment(
+      context.repo({
+        issue_number: updatedPullRequest.number,
+        body:
+          'Unassigning @author since a re-review was requested ' +
```

Comment:
This should be @{updatedPullRequest.user.login}. 

`Unassigning @{author} since a re-review was requested. @{author} Please make sure you have addressed all the review comments. Thanks!'

Reply:
Done.
Thanks